scripts/legacy/create_unique_fallback_dataset.py

```
@@ -188,50 +188,50 @@ def create_unique_fallback_dataset():
     
     # Verify no duplicates
     texts = [sample['text'] for sample in unique_samples]
-    unique_texts = set(texts)
-    print(f"🔍 Duplicate check: {len(texts)} total, {len(unique_texts)} unique")
+    unique_texts = settexts
+    print(f"🔍 Duplicate check: {lentexts} total, {lenunique_texts} unique")
     
-    if len(texts) != len(unique_texts):
-        print("❌ WARNING: DUPLICATES FOUND!")
+    if lentexts != lenunique_texts:
+        print"❌ WARNING: DUPLICATES FOUND!"
```

Ruff (0.12.2)

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected ',', found string

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

Trailing whitespace

(W291)